collections/_catalog/deployment/simple-istio-default-install.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: item
+name: Simple Istio Default Injection Installation
+type: Deployment
+compatibility:
+    - Istio
+patternId: MESHERY060
+image: /images/patterns/service-mesh.svg
+patternInfo: "This design installs the Istio service mesh version 1.12.0 control plane in the `istio-system` namespace with default profile. This requires a running Kuberenetes cluster. "
```

Comment:
@gr455 this version number will become (or has already become) outdated soon. We need to version our designs as a separate piece of metadata. The version can be user-supplied for now.